applications/registration-react/webpack.hostdev.config.js

```diff
@@ -16,15 +17,17 @@ module.exports = {
     port: 4301,
     historyApiFallback: true,
     before: app =>
-      app.get('/env.json', (_, res) =>
-        res.json({
+      app.get('/env.json', (_, res) => {
+        const defaultEnv = {
```

Comment:
Perhaps do the same as in concept-catalogue-gui?

Reply:
klar